libsentrykube/tests/test_quickpatch.py

```diff
@@ -0,0 +1,313 @@
+import os
+from typing import Generator
+from jsonschema import ValidationError
+import pytest
+from libsentrykube.context import init_cluster_context
+from libsentrykube.quickpatch import apply_patch, get_arguments
+from libsentrykube.service import (
+    get_tools_managed_service_value_overrides,
+    get_service_path,
+)
+import yaml
+import shutil
+from pathlib import Path
+
+
+def load_yaml(file_path):
+    with open(file_path, "r") as file:
+        return yaml.safe_load(file)
+
+
+SERVICE = "my_service"
+REGION = "customer1"
+TEST_PATCH = "test-patch"
+TEST_RESOURCE = "test-consumer-prod"
+CLUSTER = "default"
+TEST_NUM_REPLICAS = 10
+
+
+def print_service_dir_contents(service_dir: Path) -> None:
+    """Print all files recursively from the given directory"""
+    for root, dirs, files in os.walk(service_dir):
+        level = root.replace(str(service_dir), "").count(os.sep)
+        indent = "  " * level
+        print(f"{indent}{os.path.basename(root)}/")
+        sub_indent = "  " * (level + 1)
+        for file in files:
+            print(f"{sub_indent}{file}")
```

Comment:
Please avoid printing in your unit tests. It is not useful to provide information to the user.
If the test passes nobody looks at the output anyway also because you have to run the test with `-s`.
If the test fails, please add what is needed to the error message.